Src/EB/AMReX_EBToPVD.cpp

```diff
@@ -54,15 +54,19 @@ void EBToPVD::EBToPolygon(const Real* problo, const Real* dx,
                Real azm = apz(i  ,j  ,k  );
                Real azp = apz(i  ,j  ,k+1);
 
-               Real apnorm = std::sqrt((axm-axp)*(axm-axp) + (aym-ayp)*(aym-ayp) + (azm-azp)*(azm-azp));
+               Real adx = (axm-axp) * dx[1] * dx[2];
+               Real ady = (aym-ayp) * dx[0] * dx[2];
+               Real adz = (azm-azp) * dx[0] * dx[1];
+
+               Real apnorm = std::sqrt(adx*adx + ady*ady + adz*adz);
```

Comment:
@drangara You added this file. Could you please review the change in this file?

Reply:
The changes in this file look good to me